docs/organizations/using-a-coding-standard.md

```diff
@@ -60,7 +63,10 @@ To set a coding standard as default:
 
 1.  Open your organization **Coding standards** page, tab **Coding standards**.
 
-1.  Toggle **Make default** on the coding standard card.
+1.  Toggle **Make default** on the relevant coding standard card.
+
+    !!! note
+        Only one coding standard at a time can be the default coding standard.
 
```

Comment:
Do we want to inform users that if they accidentally turn off the make default there's no way to automatically revert to the previous status?

Reply:
If we go to this level of detail here, we should probably do the same for every other feature on the Codacy app that doesn't have a corresponding "undo" action - and that's practically all of them. :slightly_frowning_face: I wouldn't call the attention to this detail here.